packages/keystone/bin/utils.js

```diff
@@ -97,6 +97,7 @@ async function executeDefaultServer(args, entryFile, distDir, spinner) {
 
   spinner.succeed('Initialised Keystone instance');
   spinner.start('Connecting to database');
+  console.log('\n');
```

Comment:
Why is this here? It may collide with other spinner output.

Reply:
Without it the logging from Nuxt ends up on the same line as 'Connecting to database'

Reply:
Hmm 🤔 Does it interfere with the spinner from spinning correctly? Got a recording of what it looks like?

Perhaps we can insert the newline at the end of the `spinner.start`?